tests/conftest.py

```diff
@@ -211,18 +208,21 @@ def user(db):
     client = ClientDetail(
         hashed_secret="thisisatest",
         salt="thisisstillatest",
-        scopes=SCOPE_REGISTRY,
+        roles=[APPROVER],
+        scopes=[],
         user_id=user.id,
     )
 
-    FidesUserPermissions.create(
-        db=db, data={"user_id": user.id, "scopes": [PRIVACY_REQUEST_READ]}
-    )
+    FidesUserPermissions.create(db=db, data={"user_id": user.id, "roles": [APPROVER]})
```

Comment:
We had a base test user whose client had more scopes than the user did which caused a couple of tests to pass that shouldn't have due to the mismatch.  Anyway, now that users can't be assigned scopes directly, making this default user an `APPROVER` instead.